sdk/scheduler/src/test/java/com/mesosphere/sdk/offer/ResourceUtilsTest.java

```diff
@@ -0,0 +1,121 @@
+package com.mesosphere.sdk.offer;
+
+import com.mesosphere.sdk.testutils.DefaultCapabilitiesTestSuite;
+import com.mesosphere.sdk.testutils.ResourceTestUtils;
+import com.mesosphere.sdk.testutils.TestConstants;
+import org.apache.mesos.Protos;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ResourceUtilsTest extends DefaultCapabilitiesTestSuite {
```

Comment:
Technically isn't exercising resource which is missing a resource id

Reply:
Captured in a different test.